lib/active_merchant/billing/gateways/beanstream/beanstream_ipp.rb

```diff
@@ -0,0 +1,179 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class BeanstreamIppGateway < Gateway
```

Comment:
is it possible to inherit from `IppGateway`?

Reply:
I think I can do it.